src/core_modules/capture-core/components/Pages/Search/SearchDomainSelector/index.js

```diff
@@ -1 +1,2 @@
+// @flow
```

Comment:
without this here basically flow wont work :P took me a while to realise this bit